elixir/apps/web/lib/web/live/resources/edit.ex

```diff
@@ -72,7 +72,7 @@ defmodule Web.Resources.Edit do
               </p>
             </div>
 
-            <.filters_form :if={@traffic_filters_enabled?} form={@form[:filters]} />
+            <.filters_form account={@account} form={@form[:filters]} />
```

Comment:
We should also remove `traffic_filters_enabled?` assign everywhere since it's not used anymore.

Reply:
Hmm, whoops

Reply:
Updated